analyzers/src/SonarAnalyzer.CSharp/Rules/MultilineBlocksWithoutBrace.cs

```diff
@@ -28,143 +28,118 @@
 using SonarAnalyzer.Extensions;
 using SonarAnalyzer.Helpers;
 
-namespace SonarAnalyzer.Rules.CSharp
+namespace SonarAnalyzer.Rules.CSharp;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public sealed class MultilineBlocksWithoutBrace : SonarDiagnosticAnalyzer
```

Comment:
What about this? You've propose such change on other PRs
```suggestion
public sealed class MultilineBlocksWithoutBrace : SonarDiagnosticAnalyzer<SyntaxKind>
```

Reply:
If that's something we agree on also to do for rules for a one language only, I'm all in. :)

Reply:
I'm sorry, I've got confused. I thought it was already the case, but it isn't. Let's keep it as it is then